genai-perf/genai_perf/metrics/statistics_util.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from typing import List, Tuple, Union
+
+import numpy as np
+
+
+class StatisticsUtil:
+
+    @staticmethod
+    def calculate_mean(values: List[Union[int, float]]) -> float:
+        return float(np.mean(values))
```

Comment:
What is the benefit of taking these single line numpy functions out into static methods under a separate class? Before we were directly calling the numpy functions, and now we are calling static wrapper methods around the numpy functions.

Reply:
@debermudez the above is the reason why I did it. 
Open to suggestions.

Reply:
I defer to @nv-hwoo judgement here. I dont know if this is necessary now but i am not sure.

Reply:
>According to how the metrics are stored, this is the common code between the two. The way the values should be stored is different and thus, this is the design I could come up with.

In this case, if the two codes require different handling of the metrics, reducing the common code down to just a single line of numpy function, then IMO it's not really worth taking it out. This only adds extra hierarchy to the function call without any benefit of utility functions (e.g. removing duplicate code). 

+When I asked if it would be possible to reuse the `Statistics` class for the new telemetry stats class, I was wondering if we could leverage the current code in the statistics class to do the same for telemetry because they are trying to calculate the same thing. If the two functionally don't overlap, then I'm fine with the telemetrystatistics class having separate methods.